### DISCUSSION TITLE
Is there an equivalent to Soundflower Bed with BlackHole

### DISCUSSION THREAD

**Discussion by mat1696** [ExistentialAudio/BlackHole#427] [Q&A]
Hi everyone,
Blackhole is wonderful and I use it a lot to record the sound of my Mac. However I own an external monitor with internal speaker and I would like to be able to put the monitor volume always to 100% and adjust the Mac volume with the modifier keys. 
We used to be able to do that by downloading the Soundflower bed soft which worked with the old Soundflower that Blackhole is supposed to replace. So do you know if it is possible to use Blackhole the same way ? I currently use eqMac but it is a heavy and too complete soft to just enable the modifier keys…

thanks for your help !

**Comment by devinroth** [comment]
It’s really a separate app. If you search around in the GitHub issues / discussions there have been a number of solutions.

Devin Roth

**Reply by mat1696** [reply]
Thanks devinroth and sorry for the late answer.
I looked carefuly the issues and discussion tabs and didn’t find any solution/workaround… Do you have a link maybe ?

**Comment by devinroth** [comment]
How about this one?

https://github.com/ExistentialAudio/BlackHole/issues/262

Devin Roth
